frontend/config-overrides.js

```diff
@@ -9,6 +10,11 @@ const RemovePlugin = require('remove-files-webpack-plugin');
 
 const country = process.env.REACT_APP_COUNTRY;
 
+// In case GIT_HASH is not set we are in github actions environment
+process.env.REACT_APP_GIT_HASH = process.env.GITHUB_SHA
```

Comment:
this could probably become a const instead of an env at this point no?

Reply:
I don't think this code is accessible from within the application. It run before build and sets up the environment/configuration. I believe we did the same on OVIO right?